tests/runtest

```diff
@@ -0,0 +1,36 @@
+SHELL := /bin/bash
+PYTHON_VERSIONS := 3.9 3.10 3.11 3.12 3.13.2
+REQUIREMENTS := test_requirements.txt
+PYENV_EXISTS := $(shell command -v pyenv)
+
+.PHONY: all install_pyenv install_requirements install_python_versions set_global_versions run_nox
+
+all: install_pyenv install_requirements install_python_versions set_global_versions run_nox
+
+install_pyenv:
+ifeq ($(PYENV_EXISTS),)
+	@echo "Installing pyenv..."
+	@curl https://pyenv.run | bash
+else
+	@echo "Pyenv is already installed."
+endif
+
+install_requirements:
+	@echo "Installing test_requirements..."
+	@pip install -r $(REQUIREMENTS)
+	@echo "Requirements installed."
+
+install_python_versions:
+	@echo "Installing Python versions..."
+	@for version in $(PYTHON_VERSIONS); do pyenv install $$version || true; done
+	@echo "Python versions installed."
+
+set_global_versions:
```

Comment:
Is `set_global_versions` used? I'm not the best at Makefiles

Reply:
No, no, it is not a standard. This was only for local testing. Removed, thanks. [7d5c780](https://github.com/ivica-k/terraform-validate-explorer/pull/13/commits/7d5c780e3122ef97dc2b124b4a55e120de5ecc28)